### ISSUE TITLE
Call Center Add New Client function

### ISSUE THREAD

**Issue by TomOwens5000** [josephr5000/confdb-public#51]
The red * indicates fields that MUST be completed before proceeding.
If you do NOT fill in the State field you get the displayed error message.
It appears that the State field also should have the red *.
You can advance through the form WITHOUT entering something in the ZIP field.
Would it make sense (after entering Seattle in the City field to automatically populate WA in the State field? Also have a list of the cities in WA we service and populate them as WA as well?
![screen-xxx 08 08 2016](https://cloud.githubusercontent.com/assets/20777996/17492334/34e77b36-5d60-11e6-9f42-b287680ab7da.png)

**Comment by josephr5000** [comment]
A lot of questions in one issue!  Let me try to reply to each.

- Correct.  State is only required if you want to press "Locate Address", thus no red * because you might not be interested in pressing "Locate Address".  You may already know exactly where the address is located.
- Correct, ZIP isn't required.  Could you clarify why you expected it to be required?  Wondering if I'm missing something.
- Sure it would be nice to look up the city you just typed and automatically fill in the state.  The current site doesn't do that.  Nice enhancement request if you'd like to open a separate issue on that.
- Would also be nice to have a pre-populated list of cities.  Related to the enhancement request mentioned above, so you could request them together.

Closing for now as I don't think there's anything more for me to do on this ticket.  Reopen with comments if I'm mistaken about that.

**Comment by TomOwens5000** [comment]
The bit about the * was just an observation - when I tried it did 
require me to complete the State field. My comment about the ZIP field 
was wondering why it was not required if the State was required. No big 
deal here.

As mentioned earlier, I still am unable to edit notes and do not 
understand what I am missing since I think you indicated that editing 
has now been fixed and is allowed for administrators, which I am.
